code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -1607,7 +1610,12 @@ void control_config_common_read_section(int s, bool first_override) {
 				key = -1;
 			} else {
 				stuff_string(szTempBuffer, F_NAME);
-				key = mKeyNameToVal[szTempBuffer];
+				try{
```

Comment:
Please use the following brace style, it seems to be the predominant style in this .cpp
```
try {
    // code
} catch {
    // code
}
```